python/torch_mlir_e2e_test/torchscript/framework.py

```diff
@@ -157,6 +158,16 @@ def nans(self, *sizes):
         return vals
 
 
+class ResultComparisonType(Enum):
+    """ A enum class indicating how the test result is compared with the golden result.
+    """
+    # For a tensor type result, every element must be close to the golden result.
+    DEFAULT = 1
+    # For a tensor type result, the mean and standard deviation of all elements
+    # are close to the golden result.
+    STATISTIC = 2
```

Comment:
I don't think the test framework should know about this. I think it would be better to let the test itself call `torch.std`/etc. and return those values instead of the raw random values. The reason I think that is better is that it will be test-specific which statistics make sense to compare.

Reply:
Updated testing as suggested.